### DISCUSSION TITLE
Raspberry Pi Connect

### DISCUSSION THREAD

**Discussion by m3gamichi** [MichaIng/DietPi#7165] [Software Request]
Please make an server for raspberry pi connect

**Comment by MichaIng** [comment]
Try
```sh
apt install rpi-connect
```
It supports console access as well as remote desktop (based on VNC on your RPi and WebRTC between the RPi Connect website and your browser), and I am not sure whether the console access requires a desktop environment as well, if so, whether it requires the RPi desktop or whether it works with a regular LXDE, Xfce or similar, else how to configure console access. But best is to simply try it out and see what happens.

Note that this is all in beta. Since currently only a single TURN server for WebRTC in the UK is provided (seems one cannot configure an own one), it might be pretty laggy, depending on your location and how many people test it.

Ah, there is a version without remote desktop access. I.e. it must be possible to configure it from console:
```sh
apt install rpi-connect-lite
```
... and here the setup command:
```sh
rpi-connect signin
```
after you created an account.

Just read the docs: https://www.raspberrypi.com/documentation/services/connect.html
